onadata/apps/logger/models/attachment.py

```diff
@@ -86,3 +90,16 @@ def file_hash(self):
     def filename(self):
         if self.media_file:
             return os.path.basename(self.media_file.name)
+
+    @classmethod
+    def soft_delete(self, instance, user=None):
```

Comment:
I personally think the first variable here should be `cls` but that up for discussion.

Reply:
This shouldn't be a classmethod if it always requires an instance, instead it should be called off the instance w/no instance passed

Reply:
But in this case we would like access to multiple instances of `Attachment` right ? Or should this function be introduced into the `Instance` model as something like `soft_delete_attachments` ?

Reply:
ah good point! I didn't realize we're dealing w/the attachment's model, yes moving this to the `Instance` model is exactly what I was thinking